src/NUnitFramework/framework/Constraints/SubstringConstraint.cs

```diff
@@ -39,16 +44,38 @@ public SubstringConstraint(string expected) : base(expected)
         }
 
         /// <summary>
+        /// Modify the constraint to ignore case in matching.
+        /// This will call Using(StringComparison.CurrentCultureIgnoreCase) 
+        /// </summary>
```

Comment:
I think that this should have the same `InvalidOperationException` message as `Using` just so people don't have to guess how the API works.